src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -804,6 +804,10 @@ def load_arguments(self, _):
         c.argument('docker_registry_server_user', options_list=['--docker-registry-server-user', '-d'], help='The container registry server username.')
         c.argument('docker_registry_server_password', options_list=['--docker-registry-server-password', '-w'],
                    help='The container registry server password. Required for private registries.')
+        c.argument('runtime_version', options_list=['--runtime-version'],
+                   help='The runtime version for logic app.', arg_type=get_enum_type(LOGICAPPS_NODE_RUNTIME_VERSIONS))
```

Comment:
```suggestion
        c.argument('runtime_version', help='The runtime version for logic app.', arg_type=get_enum_type(LOGICAPPS_NODE_RUNTIME_VERSIONS))
```
There is no need to explicitly declare the default option name

Reply:
This is the default mapping rule of CLI core: when the `options_list` is not specified, the option name defaults to the argument name, so you can see that many parameters do not declare `options_list`